pages/linux/cpufreq-info.md

```diff
@@ -0,0 +1,35 @@
+# cpufreq-info
+
+> A tool to show CPU frequency information.
+
+- Show cpufreq kernel information for all CPUs:
+
+`cpufreq-info`
+
+- Show cpufreq kernel information for specified CPU:
+
+`cpufreq-info -c {{cpu_number}}`
+
+- Show current CPU work frequency in human-readable format, according to the cpufreq core:
```

Comment:
Perhaps `in a human-readable format`..... would read better?

Also, what is mean by `cpufreq core`?

Reply:
Actually,  `cpufreq` is the infrastructure module of implementing CPU frequency scaling in Linux kernel.
So we'll use the kernel module `cpufreq` when invoking `cpufreq-info -f` to retrieve CPU frequency, which differs from `cpufreq-info -w`  using the hardware information.

Reply:
Ah, I see - thanks 😺